src/SWP/ContentBundle/Resources/config/services.yml

```diff
@@ -1,29 +1,25 @@
 services:
-    swp_renderer.phpcr.initializer:
-        class: Doctrine\Bundle\PHPCRBundle\Initializer\GenericInitializer
-        arguments:
-            - SWP ContentBundle paths
-            - [ "/swp/content" ]
-        tags:
-            - { name: "doctrine_phpcr.initializer" }
-
     swp_template_engine.loader.article:
         class: SWP\ContentBundle\Loader\ArticleLoader
         arguments:
-            - %kernel.root_dir%
-            - @doctrine_phpcr.odm.document_manager
-            - @doctrine.orm.entity_manager
+            - "@service_container"
         tags:
             - { name: swp.meta_loader }
 
     document_manager:
         alias: doctrine_phpcr.odm.document_manager
 
-    swp_template_engine.twig.extension.content:
-        class: SWP\ContentBundle\Twig\Extension\ContentExtension
-        public: false
+    swp_content_bundle.factory.knp_paginator_representation:
+        class: SWP\ContentBundle\Factory\KnpPaginatorRepresentationFactory
         arguments:
-            - @doctrine
-            - @router
+            - 'page'
```

Comment:
it can be, but question is if we want to give option to change it to something else.